applications/luci-app-irqbalance/htdocs/luci-static/resources/view/irqbalance.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+'require view';
+'require fs';
+'require form';
+'require ui';
+'require rpc';
+
+const callServiceList = rpc.declare({
+    object: 'service',
+    method: 'list',
+    params: ['name'],
+    expect: { '': {} }
+});
+
+function getServiceStatus() {
+    return L.resolveDefault(callServiceList('irqbalance'), {}).then(function (res) {
+        try {
+            return res['irqbalance']['instances']['irqbalance']['running'];
+        } catch (e) {
+            return false;
+        }
+    });
+}
+
+function renderStatus(isRunning) {
+    const spanTemp = '%s <span style="color:%s">%s</span>';
+
+    return isRunning
+        ? String.format(spanTemp, _('Status:'), 'green', _('RUNNING'))
+        : String.format(spanTemp, _('Status:'), 'red', _('NOT RUNNING'));
+}
+
+return view.extend({
+    load() {
+        return L.resolveDefault(fs.read_direct('/proc/interrupts'));
+    },
+
+    render(data) {
+        const cpuNum = data.match(/\bCPU\d+\b/g).map(i => i.slice(3)), // cpuNum = data.match(/(?<=\bCPU)\d+\b/g), // Safari did not support RegExp lookbehind assertion before version 16.4.
+            irqNum = data.match(/\b\d+(?=: )/g);
+        let m, s, o;
+
+        m = new form.Map('irqbalance', _('irqbalance'), _('The purpose of irqbalance is to distribute hardware interrupts across processors/cores on a multiprocessor/multicore system in order to increase performance.'));
+
+        s = m.section(form.NamedSection);
+        s.anonymous = true;
+        s.render = function () {
+            L.Poll.add(function () {
+                return L.resolveDefault(getServiceStatus()).then(function (res) {
+                    let view = document.getElementById('status');
+                    view.innerHTML = renderStatus(res);
+                });
+            });
+
+            return E('div', { class: 'cbi-section' }, [
+                E('p', { id: 'status' }, _('Loading...'))
+            ]);
+        }
+
+        s = m.section(form.TypedSection, 'irqbalance', _('IRQs snapshot'));
```

Comment:
What does "IRQs snapshot" mean? Plural here might hinder genitive. Is this something like "Snapshot of current IRQs"? Please make this simpler to understand at a glance to first time users understand this quickly and translators know what to do with it.